### ISSUE TITLE
Config Check Email Sensor error

### ISSUE THREAD

**Issue by CMaxim77** [ljmerza/ha-email-sensor#11]
Hello, I installed this using HACS but I can’t set the sensors because HA says it can’t find platform email. I did install the extras via terminal as suggested in the repo. Is there Anything anybody knows what I could try?

**Comment by Stephan296** [comment]
I have got the same problem.

**Comment by CMaxim77** [comment]
Platform email not found.

Have you figured out how to get this to work @Stephan296?

**Comment by ljmerza** [comment]
where did you put files/folder in the email folder in this repo?

**Comment by Stephan296** [comment]
i put the files and folder in /custom_components/email. Like anyother custom component.

**Comment by CMaxim77** [comment]
I installed using HACS, but my files are in the same folder as Stephan’s

**Comment by CMaxim77** [comment]
Is there any suggestions that might be causing the email platform to not be found? Im running hassio docker.

I did install

pip install beautifulsoup4==4.7.1 imapclient==2.1.0 mail-parser==3.9.3, but i had to change pip to pip3 for it to install.

**Comment by CMaxim77** [comment]
I forced reset and the sensors now appear, no information yet so I’m not sure if they are working or not but it’s a good start!

**Comment by ljmerza** [comment]
do you see any logs about timing out? If the folder you specificed (indox by default) is too big then it takes a while to load and HA doesn't like that and just moves on.

**Comment by ljmerza** [comment]
Is this still an issue?

**Comment by razzam21** [comment]
I am not experiencing the issue.

--Eric Turner

**Comment by carlostico** [comment]
Installed with HACS not working : Platform error sensor.email - Integration email not found

**Comment by carlostico** [comment]
With the latest update 0.99

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 261, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 441, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/email/sensor.py", line 131, in update
    dict.fromkeys(self._attr[ATTR_TRACKING_NUMBERS][ATTR]))
KeyError: 'ups'
```

**Comment by ljmerza** [comment]
try in v2.0.1

**Comment by ljmerza** [comment]
is this still an issue?

**Comment by CMaxim77** [comment]
I never got this to work properly, unfortunately. I did get a sensor to show up after force restarting HA but it never found any tracking numbers. I might eventually give it another go. But feel free to close this issue if you would like too! I do as always appreciate all of your and other devs work on this community! I would have given up by now if it wasn’t for all the help you all provide!

**Comment by FaserF** [comment]
Just installed it with hacs.

`Platform error sensor.email - No module named 'custom_components.email.parsers.home_depot'`

**Comment by ljmerza** [comment]
fixed in 2.13.1
